queries/astro/textobjects.scm

```diff
@@ -0,0 +1 @@
+; inherits: html
```

Comment:
```suggestion
; inherits: html,tsx
```

Wdyt about inheriting also the textobjects of tsx?
An astro file does not contain html only.
Do you think the two are in conflict?

Reply:
The Astro tree-sitter parser parses only modified HTML; TypeScript and TSX are injected, so the textobjects for TS/TSX should already be handled.